dwarvesf/delivr-web
# Create transaction model and auto create when user pay online

#### Status

READY

#### What's this PR do?

- [x] Create transaction log model
- [x] Create transaction model
- [x] Auto generate transaction log when user try to pay online
- [x] Auto generate transaction when user pay online successfully
- [x] Send email to user after they pay successfully
- [x] Improve Order.show can find code with "#" before
- [x] Improve email template can show full url belong to host

#### Where should the reviewer start?

#### How should this be manually tested?

1. Make new order
2. Update total value in order record
3. Update status to 4
4. Go to /payment?code={code} to pay and check in database

#### Any background context you want to provide?

#### What are the relevant Git/YouTrack tickets?

// Put in link to Git Issue or YouTrack ticket

#### Screenshots (if appropriate)

#### Questions:

- Is there a blog post?
- Does the knowledge base need an update?
- Does this add new dependencies which need to be added to?
